exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -29,9 +29,9 @@
 
     from prometheus_client import start_http_server
 
-    from opentelemetry._metrics import get_meter_provider, set_meter_provider
+    from opentelemetry.metrics import get_meter_provider, set_meter_provider
```

Comment:
minor, but import statements are out of order

Reply:
fixed